extra/alerts/pcmk_alert_sample.sh

```diff
@@ -56,12 +61,12 @@ case $CRM_notify_kind in
 	case ${CRM_notify_desc} in
 	    Cancelled) ;;
 	    *)
-		echo "Resource operation '${CRM_notify_task}${CRM_notify_interval}' for '${CRM_notify_rsc}' on '${CRM_notify_node}': ${CRM_notify_desc}${CRM_notify_target_rc}" >> ${CRM_notify_recipient}
+		echo "${tstamp}Resource operation '${CRM_notify_task}${CRM_notify_interval}' for '${CRM_notify_rsc}' on '${CRM_notify_node}': ${CRM_notify_desc}${CRM_notify_target_rc}" >> ${CRM_notify_recipient}
 		;;
 	esac
 	;;
     *)
-        echo "Unhandled $CRM_notify_kind notification" >> ${CRM_notify_recipient}
+        echo "${tstamp}Unhandled $CRM_notify_kind notification" >> ${CRM_notify_recipient}
 	env | grep CRM_notify >> ${CRM_notify_recipient}
         ;;
 
```

Comment:
It is moved already. The comments how to use it with ClusterMon are actually still valid and ClusterMon still has options we don't cover with alerts by now (see intro above) alongside of course with disadvantages why we are doing this here. So I would put a note in front that
states how to use it with the new feature...